tests/api_fastapi/core_api/routes/ui/test_structure.py

```diff
@@ -407,7 +472,45 @@ def test_should_return_200(self, test_client, params, expected):
         assert response.status_code == 200
         assert not DeepDiff(response.json(), expected, ignore_order=True)
 
+    @pytest.mark.parametrize(
+        "params, expected",
+        [
+            pytest.param(
+                {"dag_id": DAG_ID},
+                LATEST_VERSION_DAG_RESPONSE,
+                id="get_default_version",
+            ),
+            pytest.param(
+                {"dag_id": DAG_ID, "dag_version": 1},
+                FIRST_VERSION_DAG_RESPONSE,
+                id="get_oldest_version",
+            ),
+            pytest.param(
+                {"dag_id": DAG_ID, "dag_version": 2},
+                SECOND_VERSION_DAG_RESPONSE,
+                id="get_specific_version",
+            ),
+            pytest.param(
+                {"dag_id": DAG_ID, "dag_version": 3},
+                LATEST_VERSION_DAG_RESPONSE,
+                id="get_latest_version",
+            ),
+        ],
+    )
+    @pytest.mark.usefixtures("make_dag_with_multiple_version")
+    def test_should_return_200_with_multiple_versions(self, test_client, params, expected):
+        response = test_client.get("/ui/structure/structure_data", params=params)
+        assert response.status_code == 200
+        assert not DeepDiff(response.json(), expected, ignore_order=True)
```

Comment:
Maybe don't use `DeepDiff` we use it nowhere else. Maybe a simple `response.json() == expected` is enough ?